src/libraries/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -29,7 +29,12 @@ internal sealed class ReflectionXmlSerializationReader : XmlSerializationReader
     {
         private readonly XmlMapping _mapping;
 
+        // These UnconditionalSuppressMessage attributes are understood by the trim analyzer but not by the linker.
+        // The assembly-level suppression suppresses the same warnings for the linker.
+        // https://github.com/dotnet/linker/issues/2648
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode")]
```

Comment:
Should we use pragma suppressions instead? We do that in the other places where it's "analyzer only".
I know this is slightly different, but still.